### PR TITLE
Adding a warning for VGA resolution for depth camera

### PR DESCRIPTION
**Pull request by nlyubova**
@mikaelarguedas finally, the warning is available :)

### CODE REVIEW

File: src/converters/camera.cpp [hunk 1]
```diff
@@ -99,6 +99,7 @@ const sensor_msgs::CameraInfo& getCameraInfo( int camera_source, int resolution
     if ( resolution == AL::kVGA )
     {
       static const sensor_msgs::CameraInfo cam_info_msg = createCameraInfoDEPTHVGA();
+      ROS_WARN("VGA resolution is not supported for the depth camera, please use QVGA");
```
Comment (mikaelarguedas) [inline src/converters/camera.cpp]: As this is logging it should include the the ros header with the logging macros.
Should we rephrase to say QVGA or lower ? (is QQVGA a valid resolution for the depth camera on pepper ?)

Comment (nlyubova) [reply src/converters/camera.cpp]: True that there are not much ROS_WARN in Naoqi Driver, it seems that they have preferred to use std::cout. 

What kind of macros do you mean? #include <ros/console.h> ?

Agree for "QVGA or lower"

Comment (mikaelarguedas) [reply src/converters/camera.cpp]: > What kind of macros do you mean? #include <ros/console.h> ?

yep :+1:

> Agree for "QVGA or lower"
 
:+1:

**Comment by nlyubova** [comment]
thanks! done in a new commit

File: src/converters/camera.cpp [hunk 2]
```diff
@@ -27,6 +27,7 @@
 * ROS includes
 */
 #include <cv_bridge/cv_bridge.h>
+#include <ros/console.h>
```
Comment (mikaelarguedas) [inline src/converters/camera.cpp]: I think you will have to add `rosconsole` to the package.xml as a dependency

**Comment by mikaelarguedas** [comment]
> thanks! fixed in a new commit

:+1: One more comment about dependency declaration otherwise I think this is ready to be merged!

**Comment by nlyubova** [comment]
done, in the same commit with fix.

File: package.xml [hunk 3]
```diff
@@ -28,6 +28,7 @@
   <build_depend>tf2_geometry_msgs</build_depend>
   <build_depend>tf2_msgs</build_depend>
   <build_depend>tf2_ros</build_depend>
+  <build_depend>rosconsole</build_depend>
```
Comment (mikaelarguedas) [inline package.xml]: thanks for the quick fix, one tiny comment: can you place it in alphabetical order in the list ? (it looks like all the other deps are ordered alphabetically) Same below for the run_depend

**Comment by nlyubova** [comment]
just moved it according to the order

**Comment by nlyubova** [comment]
squash ?

**Comment by mikaelarguedas** [comment]
> squash ?

Yep, you can use the "Squash and Merge" button from the github UI (arrow next to "Merge pull request")

**Pull request merged**